src/main/java/gtc_expansion/material/GTCXMaterial.java

```diff
@@ -116,7 +117,7 @@
     public static final GTMaterial Tin = new GTMaterial("Tin", 220, 220, 220, molten, smalldust, nugget, plate, stick, gear);
     public static final GTMaterial TungstenSteel = new GTMaterial(-1, "Tungstensteel", 4, 100, 100, 160, false, molten, smalldust, dust, nugget, ingot, hotIngot, plate, stick, gear, blockMetal, hull, pipeFluid);
     public static final GTMaterial Uvarovite = new GTMaterial("Uvarovite", 180, 255, 180, dustAll);
-    public static final GTMaterial Zinc = new GTMaterial(30,"Zinc", 1, 250, 240, 240, molten, smalldust, dust, nugget, ingot, plate, stick, gear, blockMetal, tinydust);
+    public static final GTMaterial Zinc = new GTMaterial(30,"Zinc", 1, 250, 240, 240, false, molten, smalldust, dust, nugget, ingot, plate, stick, gear, blockMetal, tinydust);
 
```

Comment:
Just curious: why not smeltable?

Reply:
oh shit, that shouldve been on cassiterite, dont know how i fucked that up